.pre-commit-config.yaml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
One more thing: This file is ending in `.yaml`, other YAML files in our repository end in `.yml`. Should we try to stay consistent here?

Reply:
I just tried and pre-commit expects a .yaml file, .yml is not recognized 🤷🏽‍♂️